lib/booktype/apps/loadsave/management/commands/booksave.py

```diff
@@ -14,45 +14,35 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with Booktype.  If not, see <http://www.gnu.org/licenses/>.
 
-from optparse import make_option
 from django.core.management.base import BaseCommand, CommandError
 
 from booki.editor import models
 
 
 class Command(BaseCommand):
     help = "Export book content as EPUB file."
-    args = "<book name>"
-
-    option_list = BaseCommand.option_list + (
-        make_option(
-            '--book-version',
-            action='store',
-            dest='book_version',
-            default=None,
-            help='Book version, e.g.'
-        ),
-
-        make_option(
-            '--output',
-            action='store',
-            dest='output_name',
-            default=None,
-            help='Output filename or -- for STDOUT, e.g. my_book.zip.'
-        ),
-    )
-
     requires_model_validation = False
 
-    def handle(self, *args, **options):
+    def add_arguments(self, parser):
+        parser.add_argument('<book name>', nargs=1, type=str)
+        parser.add_argument('--book-version',
+                            action='store',
+                            dest='book_version',
+                            default=None,
+                            help='Book version, e.g.')
```

Comment:
Need to give an example of book_version on line 32. Is the only possible version 1.0 since we hid the version feature?